config/custom-environment-variables.yaml

```diff
@@ -10,20 +10,18 @@ auth:
     # Generate one by running
     # $ openssl rsa -in jwt.pem -pubout -out jwt.pub
     jwtPublicKey: SECRET_JWT_PUBLIC_KEY
-    # The client id used for OAuth with github. Look up GitHub OAuth for details
-    # https://developer.github.com/v3/oauth/
-    oauthClientId: SECRET_OAUTH_CLIENT_ID
-    # The client secret used for OAuth with github
-    oauthClientSecret: SECRET_OAUTH_CLIENT_SECRET
     # The access token to use on behalf of a user to access the API. Used as
     # an alternative to the OAuth flow
     temporaryAccessKey: SECRET_ACCESS_KEY
     # The user name associated with the temproary access token. Used as a
     # means of functionally testing the API
     temporaryAccessUser: SECRET_ACCESS_USER
-    # A password used for encrypting session, and OAuth data.
+    # A password used for encrypting session data.
     # **Needs to be minimum 32 characters**
-    password: SECRET_PASSWORD
+    cookiePassword: SECRET_COOKIE_PASSWORD
```

Comment:
Have we set it up yet?
Do we need to update this? https://github.com/screwdriver-cd/kubernetes/blob/master/api.yaml